src/App.js

```diff
@@ -1,27 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <h1>Basic Calculator</h1>
```

Comment:
Great job @Kasaija-Kenneth 🤝

- [ ] Kindly make sure for this project all components are class-based. You may change this App function component into a class-based component.

Reply:
Noted, with thanks